src/Forms/Email.php

```diff
@@ -26,7 +26,7 @@ public function __construct(Submission $submission, array $config, Site $site)
         $this->submission = $submission;
         $this->config = $this->parseConfig($config);
         $this->site = $site;
-        $this->locale($site->shortLocale());
+        $this->locale($site->locale());
```

Comment:
You can use `$site->lang()` there instead, then add `'lang' => 'de_CH'` to your site config.

This way it will not be a breaking change. Site that don't have `'lang'` in their config will fall back to the short locale.

Reply:
nice idea. I've changed this and extended the tests to test this behaviour